examples/experimental/v1beta1/http-redirect-rewrite/httproute-rewrite.yaml

```diff
@@ -12,6 +12,9 @@ spec:
       - type: URLRewrite
         urlRewrite:
           hostname: elsewhere.example
+	  path:
+            type: ReplacePrefixMatch
+	    replacePrefixMatch: /fennel
```

Comment:
Although the indentation appears consistent here, it does not in the [deploy preview](https://deploy-preview-1346--kubernetes-sigs-gateway-api.netlify.app/guides/http-redirect-rewrite/). Please update this file to all use the same kind of indentation (probably spaces).

Reply:
Hi @robscott , from the deploy preview it seems that the first and second Rewrite example both got changed. So in order to rectify it , I have to create a separate file and link it in the document for the second example. Please let me know.

Reply:
Yep, you're right, thanks for catching that!

Reply:
Hi @robscott , I have created a new yaml file in the same path and linked it with the md file.Please let me know if the deploy preview looks good now and also the text for both the examples.

Reply:
Hey @subi9, thanks for the update! This looks closer but the indentation doesn't look quite right on the second example: https://deploy-preview-1346--kubernetes-sigs-gateway-api.netlify.app/guides/http-redirect-rewrite/

Reply:
Hi @robscott ,tested and fixed the indentation in yaml file.